crates/red_knot_python_semantic/src/types/display.rs

```diff
@@ -372,64 +368,8 @@ impl Display for DisplayStringLiteralType<'_> {
 
 #[cfg(test)]
 mod tests {
-    use ruff_db::files::system_path_to_file;
-    use ruff_db::system::DbWithTestSystem;
-
     use crate::db::tests::setup_db;
-    use crate::types::{global_symbol, SliceLiteralType, StringLiteralType, Type, UnionType};
-
-    #[test]
-    fn test_condense_literal_display_by_type() -> anyhow::Result<()> {
```

Comment:
IIRC we want to move unit tests that check revealed type to mdtests so I moved it to a new test called "Display of heterogeneous unions of literals" in literals.

Reply:
We don't necessarily want to move every red knot unit test to mdtest because unit tests are easier to debug, discover, and often faster, but moving this specific test makes sense.